6_2_B.py

```diff
@@ -0,0 +1,16 @@
+def get_positive_numbers():
+    """
+    Receiving input of numbers from the user
+    :return: List of positive numbers in type int
+    """
+    original_str = input("Enter numbers separated by commas: ")

```

Comment:
Seprate IO from logic - dont use input or print in functions unless its a good reason
https://wiki.c2.com/?SeparateIoFromCalculation